padrino/manifest.yml

```diff
@@ -0,0 +1,3 @@
+---
+applications:
+-  command: bundle exec padrino start -p $PORT
```

Comment:
for the record, I tried this manifest, with the same issue:
```
---
applications:
-  name: padrino-test
   command: bundle exec padrino start -p $PORT
```

but thanks for the suggestion @dhcole !